demo

```diff
@@ -35,6 +35,19 @@ while [ -n "$1" ]; do
     shift
 done
 
+# Check if the demo-artifact has been downloaded,
+# or if there exists a newer one in storage.
+DEMO_ARTIFACT_NAME="mender-demo-artifact.mender"
+wget -q --timestamping https://dgsbl4vditpls.cloudfront.net/${DEMO_ARTIFACT_NAME}
```

Comment:
Btw, one thing I forgot earlier: Should we do some quick `wget --help` and `curl --help` checks, and advice to install them if not available?